apps/transaction/forms/transaction_create_form.py

```
@@ -45,6 +97,27 @@ def save(self, commit=True):
         for debtor, share in zip(paid_for_entries, shares, strict=False):
             ChildTransaction.objects.create(parent_transaction=instance, paid_for=debtor, value=share)
 
+        self._save_receipts(instance)
+
         handle_message(ParentTransactionCreated(context_data={"parent_transaction": instance, "room": instance.room}))
 
         return instance
+
+    def _save_receipts(self, parent_transaction: ParentTransaction):
```

Ruff (0.14.5)

Missing return type annotation for private function `_save_receipts`

Add return type annotation: `None`

(ANN202)